examples/nlp/fnet_classification_with_keras_nlp.py

```diff
@@ -46,7 +46,13 @@
 """
 ## Setup
 
-Before we start with the implementation, let's import all the necessary packages.
+Before we start with the implementation, let's install and import all the
+necessary packages.
+"""
+
+"""shell
+pip uninstall -y keras-nlp
+pip install -q git+https://github.com/keras-team/keras-nlp.git@fc01b18125bb707a0de7e442a6a8e48c8fe913a3
```

Comment:
Same comment -- the problem with a line like this is that it will be out of date extremely quickly.